src/classes/UTIL_FeatureManagement_TEST.cls

```diff
@@ -0,0 +1,90 @@
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2019
+* @group Utilities
+* @group-content ../../ApexDocContent/Utilities.htm
+* @description Tests for UTIL_FeatureManagement class. Mainly for code coverage since you can't assert on
+* System.FeatureManagement method execution.
+*/
+@isTest
+public class UTIL_FeatureManagement_TEST {
+    
+    /*******************************************************************************************************
+    * @description Calls setPackageBooleanValue() method in UTIL_FeatureManagement.
+    ********************************************************************************************************/
+    @isTest
+    private static void callsSetPackageBooleanValue() { // TODO: Uncomment when completing W-033621
+        /*new UTIL_FeatureManagement().setPackageBooleanValue(
+            UTIL_OrgTelemetry.TelemetryParameterName.IsEnabled_AutomaticHouseholdNaming.name(),
+            false
+        );*/
+    }
+
+    /*******************************************************************************************************
+    * @description Calls setPackageIntegerValue() method in UTIL_FeatureManagement.
+    ********************************************************************************************************/
+    @isTest
+    private static void callsSetPackageIntegerValue() { // TODO: Uncomment when completing W-033621
+        /*new UTIL_FeatureManagement().setPackageIntegerValue(
+            UTIL_OrgTelemetry.TelemetryParameterName.Org_CountActiveCourseConnectionRecordTypes.name(),
+            3
+        );*/
+    }
+
+    /*******************************************************************************************************************
+    * @description Mock for UTIL_FeatureManagement instance
+    */
+    public class Mock implements System.StubProvider {
+        public Map<String, Boolean> packageBooleanValuesByName = new Map<String, Boolean>();
+        public Map<String, Integer> packageIntegerValuesByName = new Map<String, Integer>();
+
+        public Object handleMethodCall(
+            Object stubbedObject,
+            String stubbedMethodName,
+            Type returnType,
+            List<Type> listOfParamTypes,
+            List<String> listOfParamNames,
+            List<Object> listOfArgs
+        ) {
+            switch on stubbedMethodName {
```

Comment:
is this our very first EDA switch statement?? :)

Reply:
UTIL_OrgTelemetry.cls also has one! This is thanks to NPSP and Michael Smith, though -- I copy/pasted a lot of this directly from the Cumulus repo...